contributing/code/standards.rst

```diff
@@ -1,19 +1,33 @@
 Coding Standards
 ================
 
-When contributing code to Symfony, you must follow its coding standards. To
-make a long story short, here is the golden rule: **Imitate the existing
-Symfony code**. Most open-source Bundles and libraries used by Symfony also
-follow the same guidelines, and you should too.
+Symfony code is contributed by thousands of developers around the world. To make
+every piece of code look and feel familiar, Symfony defines some coding standards
+that all contributions must follow.
 
-Remember that the main advantage of standards is that every piece of code
-looks and feels familiar, it's not about this or that being more readable.
+These Symfony coding standards are based on the `PSR-1`_, `PSR-2`_ and `PSR-4`_
+standards, so you may already know most of them.
 
-Symfony follows the standards defined in the `PSR-0`_, `PSR-1`_, `PSR-2`_ and `PSR-4`_
-documents.
+Making your Code Follow the Coding Standards
+--------------------------------------------
 
-Since a picture - or some code - is worth a thousand words, here's a short
-example containing most features described below:
+Instead of reviewing your code manually, Symfony makes it simple to ensure that
+your contributed code matches the expected code syntax. First, install the
+`PHP CS Fixer tool`_ and then, run this command to fix any problem:
+
+.. code-block:: terminal
+
+    $ cd your-project/
+    $ php php-cs-fixer.phar fix /path/to/modified/file.php
```

Comment:
2 scenarios:
A. current state of PR:
> `$ php php-cs-fixer.phar fix /path/to/modified/file.php`

cons:
- file will be fixed even if it's excluded in config fifle
- if multiple files were fixed, dev need to provide all of them. this UX sucks and discourage user to run the command at all

B. proper command
`$ php php-cs-fixer.phar fix -v` (-v just to see changed files, progress, and so)

cons:
- user will fix whole projects, which means that he will also be fixing violations from branch he want contribute to. so even on newly cloned project, he will see violations.

-------------------------

solution
ask user to execute later command, but before asking him that - fix the branches of symfony to not report violations